app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>RecipeApp</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+    <%= stylesheet_link_tag "tailwind", "inter-font", "data-turbo-track": "reload" %>
```

Comment:
- [ ] An error is raised here when I run your application locally as seen in the screenshot below. rails db:migrate does not run your migrations. Kindly check that also from your end.
![image](https://github.com/lordksix/recipe-app/assets/62849872/2a7569a7-81aa-484f-8681-327d0e3ac679)

![image](https://github.com/lordksix/recipe-app/assets/62849872/b68a1294-cace-450f-813d-1e028124feb9)

Reply:
Thank you for the feedback. To use a tailwind base rails app you need to use `./bin/dev`. If you want to use `rails s`, you need to run first `bundle exec rake assets:precompile`

Reply:
Added instructions on the readme for future users.

Reply:
FIXED